conda_recipe/meta.yaml

```diff
@@ -123,10 +123,11 @@ outputs:
       requires:
         - jupyter
         - pytest
+        - pytest-notebook
```

Comment:
Do you need the manual deps mentioned above? Or is that only for some features?

Reply:
I'll test without the manual deps; I was expecting the conda build to fail but since it passed I may have just gotten my conda environment in an odd state.

Reply:
It is definitely possible to jam conda environment (and just occasionally the base env and hence the whole install).